src/main/scala/org/broadinstitute/dsde/workbench/sam/api/StandardSamUserDirectives.scala

```diff
@@ -29,12 +29,18 @@ trait StandardSamUserDirectives extends SamUserDirectives with LazyLogging with
   def withActiveUser(samRequestContext: SamRequestContext): Directive1[SamUser] = requireOidcHeaders.flatMap { oidcHeaders =>
     onSuccess {
       getActiveSamUser(oidcHeaders, directoryDAO, tosService, samRequestContext).unsafeToFuture()
+    }.tmap { samUser =>
+      logger.info(s"Handling request for active Sam User: $samUser")
```

Comment:
Logging the entire `SamUser` is going to create a lot of chatter. Do we need to know the accepted ToS version for each user request? Their enabled status? Their Google Subject ID or Azure B2C ID? I think it might make sense to log their email and userId, but more that that is just more log volume.